src/sentry/static/sentry/app/views/settings/account/accountSecurity/accountSecurityDetails.jsx

```diff
@@ -137,9 +131,18 @@ class AccountSecurityDetails extends AsyncView {
           action={
             authenticator.isEnrolled &&
             authenticator.removeButton && (
-              <RemoveConfirm onConfirm={this.handleRemove}>
-                <Button priority="danger">{authenticator.removeButton}</Button>
-              </RemoveConfirm>
+              <Tooltip
+                title={t(
+                  "Two-factor authentication is required for at least one organization you're a member of."
+                )}
+                disabled={!deleteDisabled}
+              >
+                <span>
```

Comment:
Is this span needed?

Reply:
`<Tooltip>` wasn't working without `<span>`. Is there another way around this?

Reply:
Nope I guess not :(